xds/internal/balancer/clusterresolver/clusterresolver.go

```diff
@@ -327,7 +327,10 @@ func (b *clusterResolverBalancer) run() {
 				b.handleClientConnUpdate(update)
 			case exitIdle:
 				if b.child == nil {
-					b.logger.Errorf("xds: received ExitIdle with no child balancer")
+					// This is not necessarily an error. The EDS/DNS watch may
+					// not have  returned a list of endpoints yet, so the child
+					// may not be built.
+					b.logger.Infof("xds: received ExitIdle with no child balancer")
```

Comment:
Either wrap this in a verbosity check of `2`. Or even completely get rid of the log message since it doesn't feel like it might be all that useful in real life.

Reply:
Wrapped it in a verbosity check.